src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -1259,9 +1356,9 @@ void GoogleCalendarSyncAdaptor::upsyncFinishedHandler()
                 // update this event in the local calendar
                 m_storage->loadNotebookIncidences(googleNotebook->uid());
                 m_storage->load(kcalEventId);
```

Comment:
Is this separate load necessary with above loadNotebookIncidences(), and if it is, should it load with recurrenceId?